packages/starknet-snap/snap.manifest.json

```diff
@@ -27,7 +27,7 @@
     ],
     "snap_manageState": {},
     "endowment:rpc": {
-      "snaps": false,
+      "snaps": true,
```

Comment:
please rollback to false, true means your snap need to talk to other snap, but we dont have such use case